### ISSUE TITLE
What is the license of files in this repo?

### ISSUE THREAD

**Issue by rob006** [extiverse/premium-translations#3]
Recently I'm trying to cleanup licenses mess for [translations repository](https://github.com/rob006-software/flarum-translations) and fetching translations from this repository is very slick in this matter, since some of these files may use proprietary license that disallows using them. Do Extiverse have some kind of requirement about releasing translations for premium extensions under permissive license (so extension may still be proprietary, but translation files are released under MIT)? In other case this repository works like a trap - it has bunch of files, which cannot be used due to its unknown license :P.

**Comment by luceos** [comment]
Good point @rob006 let me get that resolved asap. But trust me that every premium extension developer most likely will agree to MIT based license for just these translation files.

**Comment by luceos** [comment]
@rob006 pending any further objections against changes I made to the premium extension terms https://extiverse.com/terms/extension-developer this repository will soon hold the MIT license.
